docs/source/devtools/cli.md

```diff
@@ -60,11 +60,14 @@ USAGE
 
 COMMANDS
   client:check            Check a client project against a pushed service
-  client:codegen          Generate static types for GraphQL queries. Can use the published
-                          schema in Apollo Studio or a downloaded schema.
-  client:download-schema  Download a schema from Apollo Engine or a GraphQL endpoint.
+  client:codegen          Generate static types for GraphQL queries. Can use the
+                          published schema in Apollo Graph Manager or a
```

Comment:
Do you mean to change this from Apollo Studio -> Apollo Graph Manager?

Reply:
yeah oops this was me not realizing you hadn't actually published the CLI before I npx'd it :)